src/modules/settlements/settlements.test.ts

```
@@ -0,0 +1,92 @@
+import env from "@/env";
+
+import { settlementsRouter } from "./settlements.index";
+
+if (env.NODE_ENV !== "test") {
+  throw new Error("NODE_ENV must be 'test'");
+}
+
+const settlementsClient = testClient(createApp().route("/", settlementsRouter));
+
+describe("settlements", () => {
+  let userSessionToken = "";
```

ESLint

[error] 'userSessionToken' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)